lib/node.js

```diff
@@ -499,11 +482,29 @@ declare class events$EventEmitter {
   setMaxListeners(n: number): void;
 }
 
+
 declare module "events" {
   // TODO: See the comment above the events$EventEmitter declaration
   declare class EventEmitter extends events$EventEmitter {}
```

Comment:
can you change this to what you were changing it to before?

```
declare var EventEmitter : typeof events$EventEmitter;
```

that will fix the `instanceof` issue.